go/types/sequence_iterator_test.go

```diff
@@ -13,28 +13,41 @@ import (
 )
 
 var seedData = []string{
-	"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten",
+	"zero", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine",
 }
 
 const testCollectionSize = 100000
 
 func genTestBlob() (Blob, []byte) {
 	var buffer bytes.Buffer
-	for i := 0; i < testCollectionSize; i += 1 {
+	smallTestChunks()
+	defer normalProductionChunks()
+Loop:
+	for i := 0; true; i += 1 {
```

Comment:
This might be a little more clear as a straight for loop and maybe modding the count to iterate through the set.
something like this perhaps ?
for i:=0; buffer.Len() < testCollectionSize; i++ {
	v := seedData[ i % len(seedData)]
        //writeString
}